pages/dkp/kaptain/1.0.1-0.6.0/install/air-gapped/index.md

```diff
@@ -273,7 +266,12 @@ spec:
           enabled: true
         - name: traefik-forward-auth
           enabled: true
+          values: |
+            traefikForwardAuth:
+              allowedUser:
+                valueFrom:
+                  secretKeyRef: null
```

Comment:
I pushed a commit to remove Konvoy configuration snippets from Kaptain documentation because of the problems it causes during new releases. We had it planned in [D2IQ-71630](https://jira.d2iq.com/browse/D2IQ-71630) for quite some time. Instead, I've added a list of base addons required for Kaptain installation.